src/EnergyPlus/Material.cc

```diff
@@ -1710,7 +1727,7 @@ void GetMaterialData(EnergyPlusData &state, bool &ErrorsFound) // set to true if
                               state.dataIPShortCut->cAlphaFieldNames(2) + "=\"" + MaterialNames(2) +
                                   "\", must be one of DoNotModel, ModelAsDirectBeam or ModelAsDiffuse.");
         }
-        thisMaterial->Roughness = DataSurfaces::SurfaceRoughness::MediumRough;
+        thisMaterial->Roughness = Material::SurfaceRoughness::MediumRough;
```

Comment:
Again, in general don't need `namespaceX::` when you are inside that namespace.

Reply:
removed it here as well